shop/src/pages/admin/settings/legal/Edit.js

```diff
@@ -0,0 +1,321 @@
+            <fbt desc="Update">Update</fbt>
+          )
+        }
+      />
+    </div>
+  )
+
+  // Seperating out a component for CKEditor that has 'default' options set on the toolbar. To customize CKEditor's toolbar, see https://ckeditor.com/docs/ckeditor4/latest/features/toolbar.html
+  const Editor = ({ data, onChange }) => {
+    return (
+      <CKEditor
+        data={data}
+        config={{
+          toolbar: [
+            {
+              name: 'clipboard',
+              items: ['Cut', 'Copy', 'Paste', 'PasteText', '-', 'Undo', 'Redo']
+            },
+            { name: 'editing', items: ['Find', 'Replace', 'Scayt'] },
+            { name: 'forms', items: ['Button', 'ImageButton'] },
+            '/',
+            {
+              name: 'basicstyles',
+              items: [
+                'Bold',
+                'Italic',
+                'Underline',
+                'Strike',
+                'Subscript',
+                'Superscript',
+                '-',
+                'CopyFormatting',
+                'RemoveFormat'
+              ]
+            },
+            {
+              name: 'paragraph',
+              items: [
+                'NumberedList',
+                'BulletedList',
+                '-',
+                'Outdent',
+                'Indent',
+                '-',
+                'Blockquote',
+                'CreateDiv',
+                '-',
+                'JustifyLeft',
+                'JustifyCenter',
+                'JustifyRight',
+                'JustifyBlock',
+                '-',
+                'Language'
+              ]
+            },
+            { name: 'links', items: ['Link', 'Unlink'] },
+            {
+              name: 'insert',
+              items: [
+                'Image',
+                'Table',
+                'HorizontalRule',
+                'Smiley',
+                'SpecialChar'
+              ]
+            },
+            '/',
+            {
+              name: 'styles',
+              items: ['Styles', 'Format', 'Font', 'FontSize']
+            },
+            { name: 'colors', items: ['TextColor', 'BGColor'] },
+            { name: 'tools', items: ['Maximize'] }
+          ]
+        }}
+        onChange={onChange}
+      />
+    )
+  }
+
+  return (
+    <form
+      autoComplete="off"
+      onSubmit={async (e) => {
+        e.preventDefault()
+
+        if (saving) return
+
+        setSaving(true)
+
+        try {
+          // Make sure that the request body is either a JS Object or Array. Failure to do
+          // so will produce a SyntaxError when the body is parsed by Express' body-parser
+          // module (i.e. backend). Discussion: https://github.com/expressjs/body-parser/issues/309
+          const shopPoliciesRes = await post('/shop/policies', {
+            body: JSON.stringify(policies),
+            suppressError: true
+          })
+          shopPoliciesRes ? null : console.log(`Error posting shop's policies`)
```

Comment:
Shouldn't an error get displayed in the UI in case the API call returns an error?